cmd/deploy.go

```diff
@@ -492,7 +492,7 @@ func validateClusterRegistered(ctx context.Context, tenantUUID, clusterName stri
 
 	apiClient := api.NewClient(auth0Token)
 
-	if err = apiClient.PollIsClusterExist(ctx, tenantUUID, clusterName); err != nil {
+	if err = apiClient.PollIsBackendExist(ctx, tenantUUID, clusterName); err != nil {
```

Comment:
why is this still a cluster?

Reply:
what will happen if i install a cluster thats connected to an incloud deployment where cluster and backend name are not the same?

Reply:
I changed it to poll on cluster, without using backend at all..
In inCluster, users do not provide backendName (it's not part of the installation flow) so currently we can't validate installation using the backend name and we need to re-think on it.

For incloud, this flow is irrelevant - we install directly with helm charts